Assets/Scripts/GroundTracking.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+[RequireComponent(typeof(BoxCollider2D))]
+public class GroundTracking : MonoBehaviour
+{
+    private int _countGrounds;
+
+    public bool IsGround { get; private set; }
+    public int CountGrounds
+    {
+        get
+        {
+            return _countGrounds;
+        }
+        private set
+        {
+            _countGrounds = Mathf.Clamp(value, 0, int.MaxValue);
+
+            if (_countGrounds == 0)
+            {
+                if (IsGround)
+                    IsGround = false;
+            }
+            else
+            {
+                if (!IsGround)
+                    IsGround = true;
+            }
+        }
+    }
+
+    public event UnityAction<Ground> TriggerEnterGroundEvent;
```

Comment:
https://trello.com/c/CsOZfMhi/1-%D0%BD%D0%B0%D0%B7%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D0%BE%D0%B2